pkgs/build-support/wrapper-common/utils.bash

```diff
@@ -118,6 +118,19 @@ badPath() {
         "${p#"${TEMPDIR:-/tmp}"}" = "$p"
 }
 
+# Like `badPath`, but handles paths that may be interpreted relative to
+# `$SDKROOT` on Darwin. For example, `-L/usr/lib/swift` is interpreted
+# as `-L$SDKROOT/usr/lib/swift` when `$SDKROOT` is set and
+# `$SDKROOT/usr/lib/swift` exists.
+badPathWithDarwinSdk() {
+    sdkPath=$SDKROOT/$1
```

Comment:
```suggestion
    sdkPath=${SDKROOT:-}/$1
```

For `set -u` (if we ever turn it on here)

Reply:
Ah, good point. Maybe I should just move the definition inside the `if`? If `@darwinMinVersion@` is set but there’s no `SDKROOT` defined then something is wrong and the logic can’t be trusted anyway, so erroring out in that case seems good to me.

Reply:
I’ve changed this logic to hopefully be `set -u`‐proof without silently hiding `$SDKROOT` errors in the future; let me know what you think.